packages/core/src/button/Button.css

```diff
@@ -143,24 +136,18 @@
 }
 
 /* Added .uitkButton:focus for browsers don't support :focus-visible, like Firefox */
-.uitkButton:focus {
-  outline-style: var(--button-focused-outlineStyle);
-  outline-width: var(--button-focused-outlineWidth);
-  outline-color: var(--button-focused-outlineColor);
-  outline-offset: var(--button-focused-outlineOffset);
+.uitkButton:focus,
+.uitkButton:focus-visible {
```

Comment:
Don't think this will work for browsers doesn't support `:focus-visible` per [grouping spec](https://www.w3.org/TR/selectors-3/#grouping)